Formula/tiger-vnc.rb

```diff
@@ -5,6 +5,12 @@ class TigerVnc < Formula
   sha256 "9ff3f3948f2a4e8cc06ee598ee4b1096beb62094c13e0b1462bff78587bed789"
   license "GPL-2.0-or-later"
 
+  # regex here is to avoid pre-releases, like `v1.10.90`, `v1.11.90`, etc.
+  livecheck do
+    url :stable
+    regex(/^v?((?!\d+\.\d+\.9\d+)\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
  # Tags with a 90+ patch are unstable (e.g., the 1.9.90 tag is used for the
  # 1.10.0 beta release) and this regex should only match the stable versions.
  livecheck do
    url :stable
    regex(/^v?(\d+\.\d+\.(?:\d|[1-8]\d+)(?:\.\d+)*)$/i)
```

I try to avoid regex lookarounds (`(?=...)`, `(?!...)`, `(?<=...)`, `(?<!...)`) when we can, as they're usually more expensive than a comparable regex without them. This suggestion accomplishes the same goal without the lookaround.

Reply:
In this one, it does make sense. 🤞